lib/child_process.js

```diff
@@ -288,6 +292,10 @@ exports.execFile = function(file /*, args, options, callback*/) {
   return child;
 };
 
+
+exports.execFileAsync = promisify(exports.execFile);
```

Comment:
I see. Chris just mentioned a possible opt-out via flag. I'll see where this goes.